hypervisor/bsp/uefi/efi/Makefile

```diff
@@ -82,7 +72,7 @@ ifeq ($(ARCH),x86_64)
 endif
 
 LDFLAGS=-T $(LDSCRIPT) -Bsymbolic -shared -nostdlib -znocombreloc \
-		-L$(LIBDIR) $(CRT0)
+		-L$(GNUEFI_DIR) $(CRT0)
```

Comment:
It's turning out to be a little more complicated than I thought (hoped) it would be.

We cannot replace `LIBDIR` by `GNUEFI_DIR` here because altough the `LDSCRIPT` is in a `gnuefi` subdirin Fedora, the libraries it uses (`-lgnuefi -lefi`) are found in `/usr/lib` or `/usr/lib64`). So we should probably keep the logic we had for determining `LIBDIR` and combine that with the new `GNUEFI_DIR`.

I'll see if I can help you amend your patch

Reply:
I have create a Pull Request (https://github.com/zhijianli88/acrn-hypervisor/pull/1) that works for me. Can you test this in your set-up to see if that works?

The one concern left I have with my patch is that if there are multiple version of the `libgnuefi.a` and `libefi.a` libraries, we're still not being very clever about how to determine which one to use.